Framework/Kernel/src/TimeSeriesProperty.cpp

```diff
@@ -234,6 +265,66 @@ template <typename TYPE> bool TimeSeriesProperty<TYPE>::operator!=(const Propert
  */
 template <typename TYPE> void TimeSeriesProperty<TYPE>::setName(const std::string &name) { m_name = name; }
 
+/**
+ * Fill in the supplied vector of time series data according to the input TimeROI.
+ */
```

Comment:
needs `@param` entries. Also a more detailed description of which data is thrown and which retained

Reply:
Done.